messaging/src/main/java/org/axonframework/eventhandling/tokenstore/jdbc/JdbcTokenStore.java

```diff
@@ -157,6 +171,40 @@ public boolean requiresExplicitSegmentInitialization() {
         return true;
     }
 
+    @Override
+    public Optional<String> retrieveStorageIdentifier() throws UnableToRetrieveIdentifierException {
+        return Optional.of(loadConfigurationToken()).map(i -> i.get("id"));
```

Comment:
`i` refers to a `configToken`, right?

Reply:
Yes, it does. This particular method is just interested in the "id" of the config. I'll rename the variable